docs/adr/0005-feature-flags.md

```diff
@@ -0,0 +1,149 @@
+
+## Considered Options
+* Environment Variables
+* Unleashed Open Source Feature Flag (self host)
+* Unleased (managed cloud)
+
+## Decision Outcome
+
+TBD
+
+
+## Pros and Cons of the Options <!-- optional -->
+
+### Environment Variable
+
+```
+ENABLED_FEATURE_FLAGS = "premium_subplat_email, free_breaches_hibp"
+
+function isFlagEnabled(flag: string): boolean {
+  return (process.env.ENABLED_FEATURE_FLAGS ?? '').split(',').includes(flag);
+}
+
+const confettiScript = isFlagEnabled('party-mode')
+  ? '<script src="confetti.js"></script>'
+  : "";
+```
+
+* Good, because it's the easiest/quickest way to implement feature flags
+* Good, because the changes can be easily tested in separate environments, programmatically
+* Good, because the changes can be scalable (by simply adding and subtracting from the env var array)
+* Good, because it's easy to maintain and it's centralized
+* Good, because it's easy to learn and understand
+* Bad, because we will become dependent on SRE to make any changes in stage and production environment. However, it's not too bad because we will likely have to involve SRE when releasing to production anyways, and during release is when we will be deciding and modifying the environment variables. 
+* Bad, because the "implementation" is limited to the current repo. In a world where we have multiple repos serving different parts of the app, there's no easy way to share
+
+### Unleash (Cloud)
+* Good, because it is a centralized system with a dashboard that supports
+  * feature flag owner
+  * current flag state
+  * group feature flags
+  * change history
+  * expiration date and cleanup
+* Good, because set up is minimal
+* Good, because it is manged so we have support when issues occur, relatively worry-free
+* Good, because new features get added without costing us valuable engineering time
+* Bad, because we have to pay and probably sign a contract (involving legal)
+* Bad, because we adds an additonal dependency. If Unleash goes down, it will affect our service's availability
+
+
+### Unleash (self-hosted)
+* Good, because it is a centralized system with a dashboard that supports
+  * feature flag owner
+  * current flag state
+  * group feature flags
+  * change history
+  * expiration date and cleanup
+* Good, because we have visibility into the entire stack
+* Good, because it is more flexible, we can pick and choose what we need
+* Good, because it's open source and we can fork / modify features
+* Bad, because we need more support from SRE
+* Bad, because we have to keep up with open source development
```

Comment:
Question: What does upkeep look like? Making sure to install security patches/updates?

Reply:
Probably keeping an eye on it from time to time :( 
Pay more attention for major version upgrades